src/main/java/org/sunbird/cassandra/utils/CassandraOperationImpl.java

```
@@ -182,7 +182,7 @@ public void deleteRecord(String keyspaceName, String tableName, Map<String, Obje
             session.execute(updateQuery);
             response.put(Constants.RESPONSE, Constants.SUCCESS);
         } catch (Exception e) {
-            throw e;
+        	logger.error(e.getMessage());
```

Need to throw the exception - so that caller knows the issues...